bootstrap-app/src/cmr/db.clj

```diff
@@ -38,10 +38,10 @@
 
         (= "migrate" op)
         (drift/run
-         (conj
-          args
-          "-c" 
-          "config.bootstrap-migrate-config/app-migrate-config"))
+          (conj
+           (vec args)
+           "-c"
+           "config.bootstrap_migrate_config/app-migrate-config"))
```

Comment:
There are multiple changes just like this, but I don't see the matching namespaces changed to snake_case - why doesn't this break the function calls?

Reply:
These had never been changed and this is calling outside dependencies, which are named with underscores, but not translated into dashes. These classes never actually are called from within a running system